src/services/MetricsService/MetricsPusher.ts

```diff
@@ -39,6 +43,7 @@ export class DefaultMetricsPusher implements MetricsPusher {
           payload: json,
         });
       })
+      .then(() => RNSecureKeyStore.set(Key.MetricsLastUploadedDateTime, getCurrentDate().toString()))
```

Comment:
@jeberhardt  I feel like this line of code should be part of the triggerPush function in MetricsService right after we get a successful server response. All the logic for sending metrics on a daily basis would stay in the same area. What do you think?